config/routes.rb

```diff
@@ -17,4 +17,6 @@
   resources :projects do
     resources :tasks
   end
+
+  resources :project_information
```

Comment:
I think this would be better as `/project/information` rather than `/project_information`.